gtfs/spec/en/reference.md

```diff
@@ -366,7 +369,31 @@ Describe the different levels of a station. Is mostly useful when used in conjun
+| `monday` | Enum | **Conditionally Forbidden** | Indicates whether an occupancy level is valid for all Mondays in the date range specified by `occupancies.start_date` and `occupancies.end_date`. Valid options are: <br><br> `1` - The occupancy level applies for all Mondays in the date range.<br>`0` or empty - The occupancy level does not apply for all Mondays in the date range.<br><br>**Conditionally Forbidden**:<br>- **Forbidden** if `occupancies.end_date` is empty.<br>- Optional otherwise. |
+| `tuesday` | Enum| **Conditionally Forbidden** | Functions in the same way as `occupancies.monday` except applies to Tuesdays. |
+| `wednesday` | Enum| **Conditionally Forbidden** | Functions in the same way as `occupancies.monday` except applies to Wednesdays. |
+| `thursday` | Enum| **Conditionally Forbidden** | Functions in the same way as `occupancies.monday` except applies to Thursdays. |
+| `friday` | Enum| **Conditionally Forbidden** | Functions in the same way as `occupancies.monday` except applies to Fridays. |
+| `saturday` | Enum| **Conditionally Forbidden** | Functions in the same way as `occupancies.monday` except applies to Saturdays. |
+| `sunday` | Enum| **Conditionally Forbidden** | Functions in the same way as `occupancies.monday` except applies to Sundays. |
+| `start_date` | Date | **Required** | Start date of the date interval that the occupancy level is valid. |
+| `end_date` | Date | Optional | End date of the date interval that the occupancy level is valid.<br><br> If defined, `occupancies.end_date` must be greater than `occupancies.start_date`.<br><br>If empty, the occupancy level will apply only for the date specified in `occupancies.start_date`. |
```

Comment:
suggestion: what about making this **required**? If the producer wants to represent a single day, they can set `start_date` == `end_date`.